.github/workflows/test.yml

```diff
@@ -33,18 +59,10 @@ jobs:
           sudo apt-get update
           sudo apt-get install -y libgirepository1.0-dev
           python -m pip install --upgrade pip
-          pip install -e .[server] -r requirements-test.txt homeassistant
-      - name: Flake8
-        run: flake8 scripts matter_server/client matter_server/common matter_server/server
-      - name: Black
-        run: black --check scripts matter_server/client matter_server/common matter_server/server
-      - name: isort
-        run: isort --check scripts matter_server/client matter_server/common matter_server/server
-      - name: pylint
-        run: pylint matter_server/client matter_server/common matter_server/server
-      # - name: test show stored node script
-      #   run: python3 -m scripts.show_stored_node tests/fixtures/nodes/lighting-example-app.json
-      # - name: test dump fixer script
-      #   run: python3 -m scripts.dump_fixer tests/fixtures/nodes/lighting-example-app.json
-      # - name: Pytest
-      #   run: pytest tests
+          pip install -e .[server] -r requirements-test.txt
+      - name: test show stored node script
+        run: python3 -m scripts.show_stored_node tests/fixtures/nodes/lighting-example-app.json
+      - name: test dump fixer script
+        run: python3 -m scripts.dump_fixer tests/fixtures/nodes/lighting-example-app.json
```

Comment:
these should not be enabled again as they are not adjusted to the code changes

Reply:
Yeah, the whole job is disabled.